src/nikhilMJun20/UnionOfArrayLists.java

```diff
@@ -0,0 +1,41 @@
+/*
+Assignment 35:  Retrieve all unique names from given array.
+
+String[] name1 = {"Palak", "Viresh", "Yash", "Aavruti"};
+String[] name2 = {"Deavina","Palak","Viresh", "Nikhil"};
+output : [Palak, Viresh, Yash, Aavruti,Deavina,Nikhil]
+*/
+
+package nikhilMJun20;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.ListIterator;
+
+public class UnionOfArrayLists {
+	ArrayList<String> createUnion(ArrayList<String> arrList1, ArrayList<String> arrList2) {
+		//Below method will not work correctly if duplicate entries exist
+		arrList1.removeAll(arrList2);
```

Comment:
if you want to print exact order as given in question then you should,
arrList2.removeAll(arrList1);
arrList1.addAll(arrList2);

print -> arrList1